web/src/components/tasks/business-formation/contacts/Addresses.tsx

```diff
@@ -23,6 +23,7 @@ interface DisplayContent {
   snackbarHeader: string;
   snackbarBody: string;
   defaultCheckbox?: string;
+  error?: string;
```

Comment:
[dust] errorText instead of error, the name suggest the type for the prop as well.